auth-service/src/main/java/hcmus/alumni/authservice/config/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,83 @@
+package hcmus.alumni.authservice.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.web.bind.MissingServletRequestParameterException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+import org.springframework.web.multipart.MaxUploadSizeExceededException;
+import org.springframework.web.multipart.support.MissingServletRequestPartException;
+
+import hcmus.alumni.authservice.exception.AppException;
+import hcmus.alumni.authservice.exception.ErrorResponse;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+	@Value("${spring.servlet.multipart.max-file-size}")
+	private String maxFileSize;
+
+	@ExceptionHandler(Exception.class)
+	public ResponseEntity<ErrorResponse> handleGenericException(Exception e) {
+		System.out.println(e.getMessage());
+		ErrorResponse error = new ErrorResponse(500, "Lỗi hệ thống, vui lòng thử lại sau");
+		return new ResponseEntity<>(error, HttpStatus.INTERNAL_SERVER_ERROR);
+	}
+
+	@ExceptionHandler(AccessDeniedException.class)
+	@ResponseStatus(HttpStatus.FORBIDDEN)
+	public ResponseEntity<ErrorResponse> handleAccessDeniedException(AccessDeniedException ex) {
+		ErrorResponse error = new ErrorResponse(403, "Quyền truy cập bị từ chối");
+		return new ResponseEntity<>(error, HttpStatus.FORBIDDEN);
+	}
+
+	@ExceptionHandler(MissingServletRequestParameterException.class)
+	public ResponseEntity<ErrorResponse> handleMissingParams(MissingServletRequestParameterException ex) {
+		String name = ex.getParameterName();
+		ErrorResponse error = new ErrorResponse(40000, "Tham số " + name + " bị thiếu");
```

Comment:
```suggestion
		ErrorResponse error = new ErrorResponse(10000, "Tham số " + name + " bị thiếu");
```
chỉnh lại mấy cái dưới lun nha, riêng 403 với 500 ở trên vẫn để nguyên